ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -429,9 +429,14 @@ objects:
         fi
         if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
             master_type=null
-            if [[ "${CLUSTER_VARIANT}" =~ "xlarge" ]]; then
+            worker_type=m4.xlarge
+            if [[ "${CLUSTER_VARIANT}" =~ (^|,)large_worker($|,) ]]; then
```

Comment:
oh and @smarterclayton would like this to be `large-worker` (dash not underscore)